app/views/hello_world/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="alert alert-info this-works">Server Rendering</h1>
+<%= react_component("HelloWorldApp",
+                     @hello_world_props,
+                     generator_function: false,
```

Comment:
I think is already fixed, but not yet merged. I'm tempted to merge and then to generate.